templates/modules/taskbar.tpl

```diff
@@ -1,3 +1,3 @@
-<div id="taskbar" class="taskbar navbar navbar-default navbar-fixed-bottom">
```

Comment:
Right, this makes sense as the taskbar is transparent in Persona :+1:  (Note to self)